lib/binarisYML.js

```diff
@@ -60,13 +49,16 @@ const getFuncName = function getFuncName(binarisConf) {
 };
 
 const checkFuncConf = async function checkFuncConf(funcConf, funcDirPath) {
-  if (!Object.prototype.hasOwnProperty.call(funcConf, fileStr)) {
-    throw new Error(`${binarisConfFile}: function missing required field: <${fileStr}>`);
+  for (const section of [fileStr, entryStr, runtimeStr]) {
+    if (typeof funcConf[section] !== 'string') {
+      throw new Error(`${binarisConfFile}: field: <${fileStr}> should be a string`);
+    }
   }
-  if (!Object.prototype.hasOwnProperty.call(funcConf, entryStr)) {
-    throw new Error(`${binarisConfFile}: function missing required field: <${entryStr}>`);
+  const fullPath = path.join(funcDirPath, funcConf[fileStr]);
+  const stats = await fs.stat(fullPath);
+  if (!stats.isFile()) {
```

Comment:
Don't know if we need this (generally it's bad practice to check for existence outside of a read/write).

Reply:
Not related to read and write. Use, don't check. Return informational error if it fails.
(the rationale is it can fail even after checking, so the code must handle the error well, and it's better to do it once).